lstchain/tools/lstchain_create_dl2_file.py

```diff
@@ -43,6 +56,7 @@ class ReconstructionHDF5Writer(Tool):
         "energy-model": "ReconstructionHDF5Writer.path_energy_model",
         "disp-model": "ReconstructionHDF5Writer.path_disp_model",
         "gh-model": "ReconstructionHDF5Writer.path_gh_model",
+        "source-dependent": "ReconstructionHDF5Writer.source_dependent"
```

Comment:
This is a boolean, so using a flag might be better than an alias. WIth alias you have to do `--source-dependent true`, with a flag, that's just `--source-dependant`.

The format is `{flag: (config, help_string), ...}`

```
flags = {
    'source-dependant': (
        {'ReconstructionHDF5Writer': {'source_dependent': True}},
        'Perform source dependent analysis',
    )
}
```